src/files.rs

```diff
@@ -116,17 +117,17 @@ impl Files {
                 file_name
               };
               @let encoded = percent_encoding::utf8_percent_encode(&file_name, &Self::ENCODE_CHARACTERS);
-              li {
-                a href=(encoded) {
+              li class="file" {
+                a href=(encoded) class="view" {
                   (file_name)
                 }
                 @if file_type.is_file() {
-                  " - "
-                  a download href=(encoded) class=("download") {
-                    "download"
+                  a download href=(encoded) class="download" {
+                    (Files::download_icon())
                   }
                 }
               }
+              li class="separator" {}
```

Comment:
I just thought of something, I think this should be an `hr` element. There's no difference in terms of what it will look like, but `hr` is correct semantically. This is good, for example, for screen readers, which do better with semantic markup. I think this could just be `hr;`, and the CSS rule could just be `hr { … }`, i.e. it styles all `hr` elements.

Reply:
This is inside a `ul` element, where `hr`s are not allowed.

I *do* think that this separator is super hacky, so I wish we had a better solution. But I don't know one.